adding_metadata_sql.md

```diff
@@ -0,0 +1,181 @@
+## SQL
+
+SQL can also be used for bulk uploads. However, SQL should primarily be used in the following cases:
+
+* fix data errors 
+* remove duplicate records
+* replace records
+* metadata upload requires features that are not available through the web interface or BETYdb-YABA
+
+
+### Metadata Entry Workflow for BETYdb
+
+The following steps have been implemented in the BETYdb-YABA API and python client, but are described below for reference
+
+Here we show SQL code used to add metadata required each season by the TERRA REF database, including experiments, sites, treatments, cultivars, and citations that is required prior to uploading the trait or yield data. Most PEcAn users _will not_ need to add experiments and cultivars, or associate these records with sites.
+
+Also note that in the TERRA REF and other agronomic applications, each record in the 'sites' table will correspond to an experimental plot.
+
+### Step 1: Add new experiments
+
+```sql
+insert into experiments (name, start_date, end_date, user_id) 
+values ('MAC Season 6: Sorghum BAP', '2018-04-06', '2018-08-01', 'some text', 'some text', 6000000004)
+```
+
+### Step 2: Add new sites
+
+_must provide shapefile for a site to get geometry_
```

Comment:
You should either provide a full explanation (in complete sentences!) or leave this out.  And actually, thinking less "TERRA-REF-centrically", you can add simple point geometries rather easily, for example
```sql
insert into sites (city, state, country, sitename, geometry) 
values ('Urbana', 'Illinois', 'United States', 'My garden plot', ST_SetSRID(ST_makePoint(88, 40, 222), 4326));
```